docs/guides/blocks/sync-tables/dynamic.md

```diff
@@ -414,6 +414,21 @@ pack.addSyncTable({
 ```
 
 
+## Lifecycle
+
+Dynamic sync tables include a lot of individual functions, but they are run at different times and with different frequencies. Below is an outline of the typical lifecycle of a dynamic sync table.
+
+1.  **The table is expanded in the Pack's side panel**
+    - The `listDynamicUrls` function is called to populate the list of data sources.
```

Comment:
There's also the weird entrypoint where you add a dynamic table placeholder to the canvas via the slash menu, and the placeholder calls listDynamicUrls to populate a selector.

Reply:
Good callout, footnote added.